README.md

```diff
@@ -1,64 +1,229 @@
+
+If you already have existing policies in your Azure AD B2C tenant, feel free to re-use those policies in this sample.  
+
+### Step 3: Register your own Web API with Azure AD B2C
+
+As you saw in the demo environment, this sample calls a Web API at https://fabrikamb2chello.azurewebsites.net. This demo Web API uses the same code found in the sample [Node.js Web API with Azure AD B2C](https://github.com/Azure-Samples/active-directory-b2c-javascript-nodejs-webapi), in case you need to reference it for debugging purposes.
+
+You must replace the demo environment Web API with your own Web API. If you do not have your own Web API, you can clone the [Node.js Web API with Azure AD B2C](https://github.com/Azure-Samples/active-directory-b2c-javascript-nodejs-webapi) sample and register it with your tenant.
+
+#### How to setup and register the Node.js Web API sample
+
+First, clone the Node.js Web API sample repository into its own directory, for example:  
+
+```bash
+cd ..
+git clone https://github.com/Azure-Samples/active-directory-b2c-javascript-nodejs-webapi.git
+```
+
+so the two folders `active-directory-b2c-javascript-nodejs-webapi` and `active-directory-b2c-javascript-angular-spa` are located side by side.
+
+Second, follow the instructions at [register a Web API with Azure AD B2C](https://docs.microsoft.com/azure/active-directory-b2c/active-directory-b2c-app-registration#register-a-web-api) to register the Node.js Web API sample with your tenant. Registering your Web API allows you to define the scopes that your single page application will request access tokens for.
+
+Provide the following values for the Node.js Web API registration:
+
+- Provide a descriptive Name for the Node.js Web API, for example, `My Test Node.js Web API`. You will identify this application by its Name whenever working in the Azure portal.
+- Mark **Yes** for the **Web App/Web API** setting for your application.
+- Set the **Reply URL** to `http://localhost:5000`. This is the port number that the Node.js Web API sample is configured to run on.
+- Set the **AppID URI** to `hello`. This AppID URI is a unique identifier representing this Node.jS Web API. The AppID URI is used to construct the scopes that are configured in you single page application's code. For example, in this Node.js Web API sample, the scope will have the value `https://<your-tenant-name>.onmicrosoft.com/hello/demo.read`
+- Create the application.
+- Once the application is created, open your `My Test Node.js Web API` application and then open the **Published Scopes** window (in the left nav menu) and add the scope `demo.read` followed by a description `demoing a read scenario`. Click **Save**.
+
+Third, in the `index.js` file of the Node.js Web API sample, update the following variables to refer to your Web API registration.  
+
+```bash
+var tenantID = "<your-tenant-name>.onmicrosoft.com";
+var clientID = "<Application ID for your Node.js Web API - found on Properties page in Azure portal>";

```

Comment:
Can you make sure any references to client ID are `clientId` and not `clientID` for consistency, thanks!

Reply:
fixed all of 'em except the one above since it talks about the node.js web api and not this app